### DISCUSSION TITLE
Cenaze teşhis uygulaması

### DISCUSSION THREAD

**Discussion by k-kaan** [acikkaynak/deprem-yardim-projesi#43] [Ideas]
Cenazelerin isimsiz defin edilmesinden dolayı, defin öncesi bir takım bilgiler (adres, vs) ve gerekirse fotoğraf ile bu bilgiler bir veri tabanında toplanabilir ve bunu da ayriyeten ileride bir prosedür ile paylaşabilmesini sağlayabiliriz. uygulama bilgileri aldıktan sonra belirli bir sayı veya ID oluşturulur sistemde ve mezara da bu yazılır şimdilik

bir çok insan yakınını kaybediyor ve ne yazık ki hala daha ulaşılamayan insan var. sağlık açısından aciliyeti olmasa da en azından binlerce insanın isimsiz mezarlarda defnedilmesi önlenmiş olur

uygulamanın Android tarafını ben yaparım bu gece, ama backend olursa da süper olur, yoksa bilgiler sadece telefon üzerinde kalır. iOS kısmını yapacak biri olursa da birlikte hareket edebiliriz

discord: buffal0soldier#3248

**Comment by mahsumurebe** [comment]
bu işin backend tarafını ben alabilirim. fakat bunun yasal yönünü de araştırmak gerekecek. KVKK tarafından bakacak olursak, özellikle cenazelerin fotoğraflanması büyük problem yaratabilir.

**Reply by k-kaan** [reply]
Evet benim de aklimda geldi acikcasi, fotograf kisminda o yüzden emin olamadim ama enkazin adres bilgisi bile yeterli olabilir eger ki fotograf KVKK acisindan mümkün olmaz ise.

**Reply by mahsumurebe** [reply]
projenin hukuksal kısmı sorun yaratmayacaksa çok faydalı olacağını düşünüyorum. bunun için gönüllü hukukçu arkadaşlar da bilgi verirse süper olur. backend kısmını geliştirebilirim. sistemi dockerize edip ayağa kaldırılması için de destek olabilirim.

**Reply by k-kaan** [reply]
Tamam o zaman ben yarina kadar uygulamayi "offline mode" olarak ana hatlariyla hazirlayayim ve elimizde hukukculara rahatlikla gösterebilmek icin bir ürün olmus olur. 
ayni zamanda REST API icin gereksinimleri de seninle paylasirim ve detaylari birlikte hazirlariz. suan icin aklimda olan bilgilerin topluca push edilmesi internet baglantisinin kötü olmasindan dolayi

**Reply by mahsumurebe** [reply]
Süper fikir ! Gerekli REST endpointleri bildirirsen, hızlıca bir backend hazırlayacağım. Hukukçulara gösterebilmek için demo ortamı sağlarım. Bunun için sunucularımdan birinde ayağa kaldırabiliriz. Planlama kısmında detaya ihtiyaç var. Hangi bilgilerin kayıt edileceğini planlamamız gerekiyor. Bu alanları arttırıp azaltabiliriz. Verilerin gösterimi için uygulanacak prosedür ne olacak vs.

**Comment by TolgaTaner** [comment]
iOS kısmı için gönüllü olurum DC: - Tolga Taner - iOS#3606, Linkedln: https://www.linkedin.com/in/tolga-taner-62176814b/

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Fikriniz Legal ekip ile görüşmemiz sonrasında reddedilmiştir.
"Legal ret, İlgili çalışma sağlık bakanlığı bünyesinde yürütülüyor."
